Bikes/server.js

```diff
@@ -95,7 +95,7 @@ function requestIDParser(req, res, next) {
 // api ------------------------------------------------------------
 
 // find bike ------------------------------------------------------------
-app.get('/api/availableBikes', function (req, res) {
+app.get('/api/availableBikes1', function (req, res) {
```

Comment:
>1 [](start = 28, length = 1)

I hate this change it offends me on a deeply personal level REJECTED

Reply:
:smile: 

---
In reply to: [276480288](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/37#discussion_r276480288) [](ancestors = 276480288)

Reply:
Get used to it! Apparently, my demos are now about breaking code. ;-)